.github/workflows/release-app.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+    workflow_call:
+        inputs:
+            ref:
+                type: string
```

Comment:
Same as the other comment

Reply:
Here the reasoning is similar: When [releasing the latest commit to the `latest` source](https://github.com/NordicSemiconductor/pc-nrfconnect-boilerplate/blob/add_artifactory_github_actions/.github/workflows/release-latest.yml), the default ref is always correct. When [releasing manually to a specified source](https://github.com/NordicSemiconductor/pc-nrfconnect-boilerplate/blob/add_artifactory_github_actions/.github/workflows/release.yml) the default (latest commit on main) also seems like a good idea unless you explicitly want to specify something else.